docs/customresourcestate-metrics.md

```diff
@@ -480,4 +480,7 @@ Examples:
 
 # if the value to be matched is a number or boolean, the value is compared as a number or boolean  
 [status, conditions, "[value=66]", name]  # status.conditions[1].name = "b"
+
+# expand a list
```

Comment:
Can you explain a bit more what it does?

Reply:
Done below. 👍🏼